src/Raven.Server/Smuggler/Documents/SmugglerBase.cs

```diff
@@ -523,8 +523,9 @@ await foreach (var tombstone in _source.GetTombstonesAsync(_options.Collections,
                     if (tombstone.LowerId == null)
                         ThrowInvalidData();
 
-
-                    if (_options.IncludeArtificial == false && tombstone.Flags.HasFlag(DocumentFlags.Artificial))
+                    if (_options.IncludeArtificial == false && 
+                        tombstone.Flags.Contain(DocumentFlags.Artificial) && 
+                        tombstone.Flags.Contain(DocumentFlags.FromIndex))
```

Comment:
with introduction FromResharding the code can be more clean

Reply:
done